### DISCUSSION TITLE
In the wiki “Building GMP and MPFR on Windows with Visual Studio” can not find anything about “MPFR”

### DISCUSSION THREAD

**Discussion by ychaos** [CGAL/cgal#8066] [General]
Here is the url [https://github.com/CGAL/cgal/wiki/Building-GMP-and-MPFR-on-Windows-with-Visual-Studio](url)

This page didn't  refer to “MPFR”, so why the title is include “MPFR”.

**Comment by janetournois** [comment]
Hello
have you tried using the same recipe, just replacing "gmp" by "mpfr"?
